### DISCUSSION TITLE
Capstone DV

### DISCUSSION THREAD

**Discussion by gilang2287** [teamalgoritma/community#569] [Q&A] [answered]
mohon bantuannya terkait tipe data yang bisa saya ambil utk tugas capstone apa ada batas minimal jumlah data yang terlampir?
bisa di bantukah utk cthnya..

terima kasih

**Comment by finesaaa** [comment, marked as answer]
Dear pak @gilang2287,
Untuk batas minimal jumlah data tidak ada batasan spesifik angkanya pak, selama jumlah data itu mampu menjawab objective dan bussiness question tidak masalah. Misalnya saja jika Bapak melihat contoh final dashboard di rubrics, maka data yang dipakai sebenarnya tidak begitu banyak, hanya 162 row.
![image](https://user-images.githubusercontent.com/57583780/221077074-557a93c1-efd5-4945-b4dc-ef3de8e6e46c.png)
https://rpubs.com/AlgoritmaAcademy/dv-capstone
